### DISCUSSION TITLE
Mailer options - what is that for?

### DISCUSSION THREAD

**Discussion by audioscavenger** [LycheeOrg/Lychee#2320] [General]
Why did you add [Mailer options](https://lycheeorg.github.io/docs/configuration.html#mailer-options) ?

Is it for planned new ways of sharing? Or is it planned to add users emails and expand user management? 

I added my smtp mail server and all but see no way to use it within the ui, whether I am logged in or not.

**Comment by ildyria** [comment]
It is to use the notification.
In your advanced settings you can enable `new_photos_notification` and it should be sending mails every day if pictures have been added and an email has been set for the account (the email setting is only visible if notifications are enabled).

This functionality has been added since version 4.3.6
https://lycheeorg.github.io/docs/releases.html#v436
(though I do not use it).

**Comment by audioscavenger** [comment]
Thank you very much
I thought it could be used to add user's emails so I can send them login info or smth like that.
I have no use for that either, but a use case would be to know what your uploader users base do, if you have many.
